.pre-commit-config.yaml

```
@@ -7,6 +7,12 @@ repos:
         language_version: python3
     repo: https://github.com/ambv/black
     rev: 21.6b0
+  - hooks:
+      - id: codespell
+        args: 
+          - ignore-words-list ba,datas,fo,uptodate
```

I think you need to split this into two:
```suggestion
          - ignore-words-list
          - ba,datas,fo,uptodate
```

But let's see if this also works first though.